src/Atmos/Model/lsforcing.jl

```diff
@@ -0,0 +1,81 @@
+export LSForcingModel, NoLSForcing, HadGEMVertical
+
+abstract type LSForcingModel end
+
+vars_state(::LSForcingModel, ::Prognostic, FT) = @vars()
+vars_state(::LSForcingModel, ::Auxiliary, FT) = @vars()
+vars_state(::LSForcingModel, ::Gradient, FT) = @vars()
+vars_state(::LSForcingModel, ::GradientFlux, FT) = @vars()
+vars_state(::LSForcingModel, ::UpwardIntegrals, FT) = @vars()
+vars_state(::LSForcingModel, ::DownwardIntegrals, FT) = @vars()
+
+function compute_gradient_argument!(
+    lsforcing::LSForcingModel,
+    transform::Vars,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+)
+    nothing
+end
+
+function compute_gradient_flux!(
+    lsforcing::LSForcingModel,
+    diffusive::Vars,
+    ∇transform::Grad,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+)
+    nothing
+end
+
+"""
+    NoLSForcing <: LSForcingModel
+No large-scale forcing
+"""
+struct NoLSForcing <: LSForcingModel end
+
+"""
+    Container for GCM variables from HadGEM2-A forcing,
+used in the AMIP experiments
+"""
+struct HadGEMVertical <: LSForcingModel end
+
+vars_state(m::HadGEMVertical, ::Auxiliary, FT) = @vars(
+    ta::FT,
+    hus::FT,
+    ua::FT,
+    va::FT,
+    tntΣhava::FT,
+    Σtemp_tendency::FT,
+    Σqt_tendency::FT,
+    w_s::FT,
+)
+
+vars_state(::HadGEMVertical, ::Prognostic, FT) = @vars()
```

Comment:
```suggestion
```
Can let this fall back to the parent type.

Reply:
Noted, thanks !